modules/standard/CommDiagnostics.chpl

```diff
@@ -373,52 +381,52 @@ module CommDiagnostics
   /*
     Start on-the-fly reporting of communication initiated on any locale.
    */
-  proc startVerboseComm() {
+  inline proc startVerboseComm() {
```

Comment:
Yes, this makes line numbers propagate properly. Otherwise all the calls to `startVerboseComm` (and friends) have the line info of the Diagnostics module, not the user code

Reply:
> Yes, this makes line numbers propagate properly. Otherwise all the calls to startVerboseComm (and friends) have the line info of the Diagnostics module, not the user code

Did you have `CHPL_DEVELOPER=1`? That's the expected behavior if you're the developer (you shouldn't be afraid of looking at standard module code :P). OTOH, that setting shouldn't matter if you tested it with `start_test`.

Reply:
The `CHPL_DEVELOPER` differences get me all the time. without `inline` using `start_test`, I get the correct line numbers. I will remove all the inlines